content/concepts/transports/webrtc.md

```diff
@@ -101,17 +101,85 @@ Authenticity is achieved by succeeding the
 
 <!-- TO ADD DIAGRAM -->
 
-### Coming soon: Browser-to-Browser
+### WebRTC W3C (Hole punching to achieve Browser-to-Browser connectivity)
 
-Eventually, libp2p will have support for communication between two
-browsers.
+Thanks to js-libp2p and rust-libp2p (complied to Wasm), libp2p can run in the browser environment.
+However, browsers impose certain restrictions on application code (such as libp2p browser nodes).
+Applications are sandboxed and face constraints on security and networking.
+For instance, browsers do not permit direct access to raw network sockets.
+Additionally, it's a sure bet that libp2p browser nodes will be behind a NAT/firewall.
+Due to these restrictions, browser nodes cannot listen for incoming connections,
+and as a result, they cannot communicate with other browser nodes.
 
-The technical specification and initial implementations of WebRTC
-Browser-to-Browser connectivity is planned for release in early 2023.
-Track the progress [here](https://github.com/libp2p/specs/issues/475).
+Thankfully, libp2p solves this problem and enables browser node to browser node connectivity by supporting a transport called WebRTC W3C.
+
+{{< alert icon="" context="info">}}
+Due to the restrictions mentioned above, libp2p browser nodes are private nodes (meaning they cannot be dialed from outside of their local network).
+However, in principle, this use case can be thought of not just as browser-to-browser but private node to private node.
+This WebRTC solution enables connectivity between two non browser nodes behind a NAT/firewall as well
+{{< /alert >}}
+
+#### Transport Internals
+
+The libp2p WebRTC W3C transport is enabled by supporting the [W3C defined](https://w3c.github.io/webrtc-pc/#introduction) `RTCPeerConnection` API.
+This core API enables p2p connectivity and provides methods for establishing connections and transferring streams of data between peers.
+Running instances of libp2p that support this transport will have `/webrtc-w3c` in their multiaddr.
+
+However, there's more to p2p connections that what `RTCPeerConnection` provides. Crucially, signaling isn't built into the WebRTC API.
```

Comment:
It kind of is. The W3C gives you the signaling messages, you just need to figure out how to ship them to the peer yourself.

Reply:
I see, might have misrepresented what was described [here](https://web.dev/webrtc-basics/#signaling-session-control,-network,-and-media-information): "Signaling methods and protocols are not specified by WebRTC. Signaling is not part of the RTCPeerConnection API."

I'll take a note out of this: https://webrtc.org/getting-started/peer-connections#signaling and reword as:
"Crucially, the WebRTC specification doesn't prescribe any single solution for signaling. The peer connection API has a notion of [signaling state](https://w3c.github.io/webrtc-pc/#dom-rtcsignalingstate) but not mechanism to share this data with remote peers."

lmk what you think